src/test/java/com/endava/soa4/RunCucumberTest.java

```diff
@@ -0,0 +1,14 @@
+package com.endava.soa4;
+
+import io.cucumber.junit.Cucumber;
+import io.cucumber.junit.CucumberOptions;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        features = "resources/feature",
+        glue = {"java/com/endava/soa4/stepDefinition"},
+        tags = "@Claudia and @Irina and @Stefan and @Artiom and @Anatolie")
```

Comment:
This means that this Runner will run only tests with all this tags present. Please change this, as you need to be present only one name and @Run tag

Reply:
Done